.kitchen.yml

```diff
@@ -64,8 +64,6 @@ suites:
     attributes:
       audit:
         profiles: &profiles
```

Comment:
leftover from initial implementation.. Yaml syntax docs: "Repeated nodes are first identified by an anchor (marked with the ampersand - “&”), and are then aliased (referenced with an asterisk - “*”) thereafter."